corehq/sql_proxy_accessors/migrations/0045_form_attachments.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2018-11-17 20:58
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq.sql_db.operations import RawSQLMigration, SQL
+
+migrator = RawSQLMigration(('corehq', 'sql_proxy_accessors', 'sql_templates'))
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sql_proxy_accessors', '0044_blobmeta_form_attachments'),
+    ]
+
+    operations = [
+        migrator.get_sql(
+            SQL("DROP FUNCTION IF EXISTS get_form_attachment_by_name(TEXT, TEXT)"),
+            "get_form_attachment_by_name.sql",
+        ),
+        migrator.get_sql(
+            SQL("DROP FUNCTION IF EXISTS get_form_attachments(TEXT)"),
+            "get_form_attachments.sql",
+        ),
+        migrator.get_sql(
+            SQL("DROP FUNCTION IF EXISTS get_multiple_forms_attachments(TEXT[])"),
+            "get_multiple_forms_attachments.sql",
```

Comment:
Guessing removing the query files will come in a later PR?

Reply:
Technically they should stick around as long as there are any migrations referencing them. In the past we haven't done this very consistently, and it's one of the reasons our migrations break for people who do not keep their database on the cutting edge of master.